Resources/Prototypes/Nyanotrasen/Recipes/Cooking/recipes_moth.yml

```diff
@@ -0,0 +1,331 @@
+# Moth related cuisine
+# A lot of those recipes are using placeholders ingredients
+
+- type: microwaveMealRecipe
+  id: RecipeMothToastedSeeds
+  name: toasted seeds recipe
+  result: FoodMothToastedSeeds
+  time: 15
+  reagents:
+    OilOlive: 5
+  solids:
+    WheatSeeds: 1
+    CarrotSeeds: 1
+    PoppySeeds: 1
+
+- type: microwaveMealRecipe
+  id: RecipeMothBakedCorn
+  name: baked corn recipe
+  result: FoodMothBakedCorn
+  time: 20
+  solids:
+    FoodCorn: 1
+
+- type: microwaveMealRecipe
+  id: RecipeMothButteredBakedCorn
+  name: buttered baked corn recipe
+  result: FoodMothButteredBakedCorn
+  time: 15
```

Comment:
Just 5 secs should be enough to melt a bit of butter, let's be nice.

Reply:
Reduced BakedCorn to 15s and ButteredBakedCorn to 5s so it takes 20 in total